joss_paper/paper.md

```diff
@@ -0,0 +1,167 @@
+---
+title: 'ndcube: Manipulating N-dimensional Astronomical Data in Python'
+tags:
+  - Python
+  - astronomy
+authors:
+  - name: Daniel F. Ryan^[co-first author] # note this makes a footnote saying 'co-first author'
+    orcid: 0000-0001-8661-3825
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Stuart Mumford^[co-first author] # note this makes a footnote saying 'co-first author'
+    affiliation: 2
+  - name: Author with no affiliation^[corresponding author]
+    affiliation: 3
+affiliations:
+ - name: University of Applied Sciences Northwest Switzerland
+   index: 1
+ - name: University of Sheffield
+   index: 2
+date: 11 December 2021
+bibliography: paper.bib
+
+---
+
+# Summary
+
+ndcube is a free, open-source, community-developed Python package whose purpose
+is to faciliate the manipulation and visualization of astronomical data.
+It does this by linking the data and coordinates in single data objects.
+These objects can be manipulated via array-like slicing operations which modify
+the data and coordinates simultaneously.
+They also allow coordinate transformations to be performed with reference to the
+shape of the data and enable coordinate-aware visualizations.
+This data-coordinate coupling allows users to analyze their data more
+easily and accurately, thus helping to boost their scientific output.
```

Comment:
I think we should probably mention, and reference, Astropy's APE 14 here, that in reality is what makes us unique among the wider scientific python ecosystem. https://zenodo.org/record/1188875